cachito/web/models.py

```diff
@@ -373,6 +376,10 @@ def to_json(self, verbose=True):
         # Show the latest state information in the first level of the JSON
         rv.update(latest_state)
         if verbose:
+            if self.submitted_by:
```

Comment:
Can we always display this, instead of just when verbose mode?
Not doing so may cause confusion.

Reply:
Sure. I see it as more of an implementation detail which is why I thought it'd be hidden to just verbose output, but I'll change that.